client/blocks/support-article-dialog/dialog.jsx

```diff
@@ -47,6 +48,11 @@ export const SupportArticleDialog = ( {
 	const siteId = post?.site_ID;
 	const shouldQueryReaderPost = ! post && ! shouldRequestAlternates && ! isRequestingAlternates;
 
+	const [ supportArticleId, updateSupportArticle ] = useUrlSearchQueryState( 'support-article' );
+	if ( ! actionUrl ) {
+		actionUrl = 'https://support.wordpress.com?p=' + supportArticleId;
```

Comment:
Let's avoid mutating incoming props. This can lead to unexpected bugs sometimes. By all means, let's use another constant and keep things immutable.

Reply:
Should be fixed now.